### PR TITLE
Remove aws-sdk dependency

### PR DESCRIPTION
**Pull request by dcousens**
... and fixes the failing tests

Related
- https://github.com/keystonejs/keystone/issues/8023

### CODE REVIEW

File: index.test.ts [hunk 1]
```diff
@@ -8,188 +8,147 @@ import retry from 'async-retry';
...
-        page.setDefaultNavigationTimeout(6000);
-        await page.goto('http://localhost:3000');
-      });
+      await deleteAllData(projectDir);
+      browser = await playwright[browserName].launch();
+      page = await browser.newPage();
+      page.setDefaultNavigationTimeout(6000);
     });
-    test('init user', async () => {
-      await retry(async () => {
```
Comment (dcousens) [inline index.test.ts:142]: retry is unnecessary

File: index.test.ts [hunk 2]
```diff
@@ -8,188 +8,147 @@ import retry from 'async-retry';
...
-          'label:has-text("Email") >> .. >> input',
-          'admin@keystonejs.com'
-        );
-        await page.click('button:has-text("Set Password")');
-        await page.fill('[placeholder="New Password"]', 'password');
-        await page.fill('[placeholder="Confirm Password"]', 'password');
-        await page.click('button:has-text("Get started")');
-        await page.uncheck('input[type="checkbox"]', { force: true });
-        await Promise.all([
-          page.waitForNavigation(),
```
Comment (dcousens) [inline index.test.ts:154]: `waitForNavigation` is unnecessary when `click` automatically waits for the selector

File: package.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
     "lint:prettier": "prettier --list-different \"**/*.{js,json,ts,tsx,md}\"",
     "lint:types": "tsc",
     "test": "yarn lint:prettier && yarn lint:types",
-    "test:unit": "DATABASE_URL=file:./keystone.db yarn jest",
+    "test:unit": "DATABASE_URL=file:./keystone.db yarn jest -- --runInBand",
```
Comment (dcousens) [inline package.json:37]: This, this was the primary problem

File: index.test.ts [hunk 4]
```diff
@@ -8,188 +8,150 @@ import retry from 'async-retry';
...
-        ]);
-        await page.type('input[type="text"]', '1');
-        await page.click('button:has-text("Save changes")');
-      });
+    test('create a post', async () => {
+      await page.click('nav >> text=Posts');
+      await page.click('a:has-text("Create Post")');
+      await page.fill('input[type="text"]', 'title');
+      await page.click('button:has-text("Create Post")');
+      await page.waitForTimeout(2000); // sleep for a second or two
```
Comment (dcousens) [inline index.test.ts:150]: The problem is `Save changes` isn't actually ready, could probably fix that later - this is fine for now

File: package.json (hunk 3 above)
Comment (dcousens) [inline package.json:37]: Not strictly a reason for the broken tests, but, technically its a requirement that we rely on the tests running in sequence.

File: create-keystone-app/starter/auth.ts [hunk 5]
```diff
@@ -39,20 +39,17 @@ const { withAuth } = createAuth({
   sessionData: 'name createdAt',
   secretField: 'password',
 
-  // dont support initFirstItem functionality in production
+  // WARNING: remove initFirstItem functionality in production
   //   see https://keystonejs.com/docs/config/auth#init-first-item for more
-  initFirstItem:
-    process.env.NODE_ENV === 'production'
```
Comment (dcousens) [inline create-keystone-app/starter/auth.ts:45]: Unfortunate, but we will remove this soon enough

File: index.test.ts [hunk 6]
```diff
@@ -129,67 +110,48 @@ describe.each(['development', 'production'] as const)('%s', (mode) => {
...
-        ]);
-        await page.type('input[type="text"]', '1');
-        await page.click('button:has-text("Save changes")');
-      });
+    test('create a post', async () => {
+      await page.click('nav >> text=Posts');
+      await page.click('a:has-text("Create Post")');
+      await page.fill('input[type="text"]', 'title');
+      await page.click('button:has-text("Create Post")');
+//        await page.waitForTimeout(2000); // TODO: the 'Save changes' button is painful
```
Comment (dcousens) [inline index.test.ts:150]: Sigh

File: index.test.ts [hunk 7]
```diff
@@ -129,67 +110,48 @@ describe.each(['development', 'production'] as const)('%s', (mode) => {
...
-        ]);
-        await page.type('input[type="text"]', '1');
-        await page.click('button:has-text("Save changes")');
-      });
+    test('create a post', async () => {
+      await page.click('nav >> text=Posts');
+      await page.click('a:has-text("Create Post")');
+      await page.fill('input[type="text"]', 'title');
+      await page.click('button:has-text("Create Post")');
+      //        await page.waitForTimeout(2000); // TODO: the 'Save changes' button is painful
```
Comment (dcousens) [inline index.test.ts:150]: Sigh

**Review by mitchellhamilton — APPROVED** [review]

**Pull request merged**